Description: Adjust menu location of progam
 Add more categories to the desktop file to adjust where
 Cura show up in the GUI menu.
Author: Petter Reinholdtsen <pere@hungry.com>
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2017-10-08

--- cura-2.5.0.orig/cura.desktop.in
+++ cura-2.5.0/cura.desktop.in
@@ -12,5 +12,5 @@ Icon=@CMAKE_INSTALL_FULL_DATADIR@/cura/r
 Terminal=false
 Type=Application
 MimeType=application/sla;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;image/bmp;image/gif;image/jpeg;image/png
-Categories=Graphics;
+Categories=Graphics;Education;Development;Science;
 Keywords=3D;Printing;
